addons/website_slides/static/src/js/slides_slide_archive.js

```diff
@@ -30,6 +30,17 @@ var SlideArchiveDialog = Dialog.extend({
         this.slideId = this.$slideTarget.data('slideId');
         this._super(parent, options);
     },
+    _displayCategoryEmptyFlag: function (){
```

Comment:
Rename `_checkForEmptySections` for coherence.
And also check if any part of my previous comment applies here too.